src/tests/unit/tests/electron/platform/android/__snapshots__/scan-results-to-unified-results.test.ts.snap

```diff
@@ -4,43 +4,86 @@ exports[`ScanResultsToUnifiedResults Null ScanResults input returns empty output
 
 exports[`ScanResultsToUnifiedResults ScanResults with no RuleResults returns empty output 1`] = `Array []`;
 
-exports[`ScanResultsToUnifiedResults ScanResults with passes, failures, and excluded results 1`] = `
+exports[`ScanResultsToUnifiedResults ScanResults with passes, failures, view IDs, and excluded results 1`] = `
 Array [
   Object {
-    "descriptors": null,
+    "descriptors": Object {
+      "boundingRectangle": Object {
+        "bottom": 800,
+        "left": 10,
+        "right": 600,
+        "top": 0,
+      },
+      "className": "myClass1",
+      "contentDescription": "myDescription1",
+      "text": "myText1",
+    },
     "identifiers": null,
     "resolution": Object {
-      "how-to-fix": "The text element has insufficient contrast of 1. Foreground color: #ffffff, background color: #ffffff). Modify the text foreground and/or background colors to provide a contrast ratio of at least 4.5:1 for regular text, or 3:1 for large text (at least 18pt, or 14pt+bold).",
+      "how-to-fix": Object {
+        "unformattedText": "The text element has insufficient contrast of 1. Foreground color: #ffffff, background color: #ffffff). Modify the text foreground and/or background colors to provide a contrast ratio of at least 4.5:1 for regular text, or 3:1 for large text (at least 18pt, or 14pt+bold).",
+      },
     },
     "ruleId": "ColorContrast",
     "status": "fail",
     "uid": "gguid-mock-stub",
   },
   Object {
-    "descriptors": null,
+    "descriptors": Object {
+      "boundingRectangle": null,
+      "className": "myClass3",
+      "contentDescription": null,
+      "text": null,
```

Comment:
What's the consensus on returning null values in properites? Is it preferable to simply omit the keys if the values are null?

Reply:
I think this is fine... the class using `descriptors` needs to handle the `text` being `null`